run.py

```diff
@@ -74,6 +74,14 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10):
             torch.cuda.synchronize()
             t1 = time.time_ns()
 
+            result_summary.append((start_event.elapsed_time(end_event), (t1 - t0) / 1_000_000))
+        elif args.device == "mps":
+            t0 = time.time_ns()
+            func()
+            t1 = time.time_ns()
+            wall_latency = t1 - t0
+            # modify this to add GPU time as well
```

Comment:
Is this an TODO item? Can we add a `TODO:` prefix to the line 83 comment?

Reply:
Done.